.github/workflows/macos-executable-release.yml

```diff
@@ -28,9 +28,28 @@ jobs:
         run: |
           git fetch --prune --unshallow
           echo "::set-output name=SHA_SHORT::$(git rev-parse --short HEAD)"
-  pack-macos:
+  set-file-path:
     needs: [get-tag, get-sha]
     runs-on: macos-11
+    outputs:
+     FILE_PATH_X64: ${{steps.set-file-path.outputs.FILE_PATH_X64}}
+    steps:
+     - uses: actions/checkout@v2
+     - name: Setting file path
+       id: set-file-path
+       run: |
+          if [ "${{ github.ref_name }}" == "main" ]; then
+            file_path_x64=dist/macos/${{ github.event.inputs.formula }}-v${{ needs.get-tag.outputs.TAG_NAME }}.pkg
+            echo "FILE_PATH_X64=$file_path_x64" >> $GITHUB_OUTPUT
+          else
+            file_path_x64=dist/macos/${{ github.event.inputs.formula }}-v${{ needs.get-tag.outputs.TAG_NAME }}.${{ needs.get-sha.outputs.SHA_SHORT }}.pkg
```

Comment:
This else condition would be applicable for 2 types of releases: rc and draft. Do we need this naming convention for both of them?

Reply:
Changed it to be for rc only